dmutils/documents.py

```diff
@@ -190,6 +190,16 @@ def get_agreement_document_path(framework_slug, supplier_id, supplier_name, docu
     )
 
 
+def get_countersigned_agreement_document_path(framework_slug,
+                                              supplier_id,
+                                              document_name="countersigned-framework-agreement.pdf"):
```

Comment:
I thought about this but I don't think CCS will countersign things as a JPEG.  But I'll check to be sure.